### PR TITLE
Thellimist/pi 717

### PR DESCRIPTION
**Pull request by thellimist**

### CODE REVIEW

File: src/WordPress/DataStore.php [hunk 1]
```diff
@@ -68,7 +68,7 @@ public function getClientV4APIKey()
      */
     public function getHostAPIUserKey()
     {
-        return null;
+        return;
```
Comment (jwineman) [inline src/WordPress/DataStore.php]: why did you remove the `null` here?

Comment (thellimist) [inline src/WordPress/DataStore.php]: commit c554533 is done automatically by php code sniffer. I set sublime to run `phpcs -n --standard=PSR2` in every save for php files. 

https://github.com/php-fig-rectified/fig-rectified-standards/blob/master/PSR-2-R-coding-style-guide-additions.md#return-void-vs-null
